### ISSUE TITLE
Could not load type 'System.ComponentModel.AsyncCompletedEventArgs'

### ISSUE THREAD

**Issue by joaosantana** [GitTools/GitVersion#1873]
Fresh repo, fresh project, receiving this error when building in AppVeyor:
```
gitversion /l console /output buildserver /updateAssemblyInfo
INFO [10/28/19 17:31:25:93] Working directory: C:\projects\recibos
INFO [10/28/19 17:31:25:94] IsDynamicGitRepository: False
INFO [10/28/19 17:31:25:97] Returning Project Root from DotGitDirectory: C:\projects\recibos\.git\ - C:\projects\recibos\
INFO [10/28/19 17:31:25:97] Running on Windows.
INFO [10/28/19 17:31:25:97] Applicable build agent found: 'AppVeyor'.
INFO [10/28/19 17:31:25:97] Branch from build environment: master
INFO [10/28/19 17:31:25:98] Begin: Normalizing git directory for branch 'master'
INFO [10/28/19 17:31:26:01] One remote found (origin -> 'https://github.com/joaosantana/recibos.git').
INFO [10/28/19 17:31:26:02] Fetching from remote 'origin' using the following refspecs: +refs/heads/*:refs/remotes/origin/*.
INFO [10/28/19 17:31:26:19] Updating local branch refs/heads/master to match ref master
INFO [10/28/19 17:31:26:23] Creating local branch from remote tracking 'refs/remotes/origin/develop'.
INFO [10/28/19 17:31:26:25] HEAD points at branch 'refs/heads/master'.
INFO [10/28/19 17:31:26:25] End: Normalizing git directory for branch 'master' (Took: 274.59ms)
INFO [10/28/19 17:31:26:25] IsDynamicGitRepository: False
INFO [10/28/19 17:31:26:25] Returning Project Root from DotGitDirectory: C:\projects\recibos\.git\ - C:\projects\recibos\
INFO [10/28/19 17:31:26:25] Project root is: C:\projects\recibos\
INFO [10/28/19 17:31:26:25] DotGit directory is: C:\projects\recibos\.git
INFO [10/28/19 17:31:26:29] IsDynamicGitRepository: False
INFO [10/28/19 17:31:26:29] Returning Project Root from DotGitDirectory: C:\projects\recibos\.git\ - C:\projects\recibos\
INFO [10/28/19 17:31:26:30] Begin: Loading version variables from disk cache
INFO [10/28/19 17:31:26:30] Cache file C:\projects\recibos\.git\gitversion_cache\AD36760B0CBEB73CF54BAFA08A79116F03AC5413.yml not found.
INFO [10/28/19 17:31:26:30] End: Loading version variables from disk cache (Took: 3.93ms)
INFO [10/28/19 17:31:26:30] IsDynamicGitRepository: False
INFO [10/28/19 17:31:26:31] Returning Project Root from DotGitDirectory: C:\projects\recibos\.git\ - C:\projects\recibos\
INFO [10/28/19 17:31:26:46] Using latest commit on specified branch
INFO [10/28/19 17:31:26:48] Running against branch: master (b8b74302b6295f13db412eea5a98d5e4aa831882)
INFO [10/28/19 17:31:26:48] Begin: Calculating base versions
INFO [10/28/19 17:31:26:49] Fallback base version: 0.1.0 with commit count source b8b74302b6295f13db412eea5a98d5e4aa831882 (Incremented: None)
INFO [10/28/19 17:31:26:53] Found multiple base versions which will produce the same SemVer (0.1.0), taking oldest source for commit counting (Fallback base version)
INFO [10/28/19 17:31:26:53] Base version used: Fallback base version: 0.1.0 with commit count source b8b74302b6295f13db412eea5a98d5e4aa831882 (Incremented: None)
INFO [10/28/19 17:31:26:53] End: Calculating base versions (Took: 48.55ms)
INFO [10/28/19 17:31:26:53] Skipping version increment
INFO [10/28/19 17:31:26:53] 0 commits found between b8b74302b6295f13db412eea5a98d5e4aa831882 and b8b74302b6295f13db412eea5a98d5e4aa831882
INFO [10/28/19 17:31:26:53] Begin: Getting version tags from branch 'refs/heads/master'.
INFO [10/28/19 17:31:26:54] End: Getting version tags from branch 'refs/heads/master'. (Took: 4.16ms)
INFO [10/28/19 17:31:26:55] Begin: Creating dictionary
INFO [10/28/19 17:31:26:55] End: Creating dictionary (Took: 7.12ms)
INFO [10/28/19 17:31:26:56] Begin: Storing version variables to cache file C:\projects\recibos\.git\gitversion_cache\AD36760B0CBEB73CF54BAFA08A79116F03AC5413.yml
INFO [10/28/19 17:31:26:63] End: Storing version variables to cache file C:\projects\recibos\.git\gitversion_cache\AD36760B0CBEB73CF54BAFA08A79116F03AC5413.yml (Took: 71.02ms)
INFO [10/28/19 17:31:26:63] Applicable build agent found: 'AppVeyor'.
Executing GenerateSetVersionMessage for 'AppVeyor'.
Set AppVeyor build number to '0.1.0-stable.1+0'.
Executing GenerateBuildLogOutput for 'AppVeyor'.
gitversion :   ERROR [10/28/19 17:31:27:84] An unexpected error occurred:
At line:1 char:1
+ gitversion /l console /output buildserver /updateAssemblyInfo
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (  ERROR [10/28/...error occurred::String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
System.TypeLoadException: Could not load type 'System.ComponentModel.AsyncCompletedEventArgs' from assembly 'System.Net.WebHeaderCollection, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f
5f7f11d50a3a'.
   at GitVersion.BuildServers.AppVeyor.GenerateSetParameterMessage(String name, String value)
   at GitVersion.OutputFormatters.BuildOutputFormatter.GenerateBuildLogOutput(IBuildServer buildServer, VersionVariables variables)
   at GitVersion.BuildServerBase.WriteIntegration(Action`1 writer, VersionVariables variables)
   at GitVersion.ExecCommand.Execute()
   at GitVersion.GitVersionExecutor.VerifyArgumentsAndRun(Arguments arguments)
INFO [10/28/19 17:31:27:84] 
INFO [10/28/19 17:31:27:84] Attempting to show the current git graph (please include in issue): 
INFO [10/28/19 17:31:27:84] Showing max of 100 commits
INFO [10/28/19 17:31:28:00] * b8b7430 2 minutes ago  (HEAD -> master, origin/master, origin/develop, origin/HEAD, develop)
```

My ```GitVersion.yml``` file:

```
mode: ContinuousDelivery
branches:
  develop:
    tag: dev
  feature:
    tag: feat
  hotfix:
    tag: fix
  master:
    tag: stable
  release:
    tag: rc
  pull-request:
    tag: pr
  support:
    tag: support
ignore:
  sha: []
merge-message-formats: {}
```

**Comment by asbjornu** [comment]
Is this with version 5.1.1 of GitVersion?

**Comment by Caleb9** [comment]
@asbjornu I'm experiencing the same issue and yes, it's version 5.1.1 installed through chocolatey (we're printing out GitVersion's version for diagnostics, see below).

```
gitversion /version
5.1.1+Branch.master.Sha.5d79a7475c182b8b9e75090fd40fdd3c1157e302
echo *** Calculating semantic version ***
*** Calculating semantic version ***
gitversion /verbosity warn /l console /output buildserver /nofetch /b %APPVEYOR_REPO_BRANCH%
Executing GenerateSetVersionMessage for 'AppVeyor'.
Set AppVeyor build number to '2.1.0-gitversion-update.1+17'.
Executing GenerateBuildLogOutput for 'AppVeyor'.
  ERROR [10/29/19 9:53:21:13] An unexpected error occurred:
System.TypeLoadException: Could not load type 'System.ComponentModel.AsyncCompletedEventArgs' from assembly 'System.Net.WebHeaderCollection, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
   at GitVersion.BuildServers.AppVeyor.GenerateSetParameterMessage(String name, String value)
   at GitVersion.OutputFormatters.BuildOutputFormatter.GenerateBuildLogOutput(IBuildServer buildServer, VersionVariables variables)
   at GitVersion.BuildServerBase.WriteIntegration(Action`1 writer, VersionVariables variables)
   at GitVersion.ExecCommand.Execute()
   at GitVersion.GitVersionExecutor.VerifyArgumentsAndRun(Arguments arguments)
  ERROR [10/29/19 9:53:21:13] An unexpected error occurred:
System.TypeLoadException: Could not load type 'System.ComponentModel.AsyncCompletedEventArgs' from assembly 'System.Net.WebHeaderCollection, Version=4.1.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
   at GitVersion.BuildServers.AppVeyor.GenerateSetParameterMessage(String name, String value)
   at GitVersion.OutputFormatters.BuildOutputFormatter.GenerateBuildLogOutput(IBuildServer buildServer, VersionVariables variables)
   at GitVersion.BuildServerBase.WriteIntegration(Action`1 writer, VersionVariables variables)
   at GitVersion.ExecCommand.Execute()
   at GitVersion.GitVersionExecutor.VerifyArgumentsAndRun(Arguments arguments)
```

**Comment by joaosantana** [comment]
> Is this with version 5.1.1 of GitVersion?

Yes, installed via Chocolatey, as @Caleb9 commented out.

**Comment by Caleb9** [comment]
Still happening on 5.1.2.

**Comment by warning-explosive** [comment]
Works fine with GitVersion 4.0.0 in AppVeyor build

**Comment by skwasjer** [comment]
I've tested a bit with AppVeyor.

Note: I've omitted AspNetCore runtimes

image: Visual Studio 2017

.NET Core runtimes installed:
  Microsoft.NETCore.App 1.0.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 1.0.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 1.0.16 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 1.1.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 1.1.2 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 1.1.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.0.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.2 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.10 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
image: Visual Studio 2019

.NET Core runtimes installed:
  Microsoft.NETCore.App 1.0.16 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 1.1.13 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.10 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.11 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.1 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.4 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
The issue I believe has to do with the dependency System.ComponentModel.EventBasedAsync.dll missing in GitVersion's published package. Thus, the assembly will be resolved from preinstalled runtimes.

The dependency graph is:

GitVersionCore -> System.Net.WebClient 4.0.1.0 -> System.ComponentModel.EventBasedAsync.dll 4.1.0.0
Version 4.1.0.0 however is only available up until runtime Microsoft.NETCore.App 2.0.9. This works on the VS2017 AppVeyor image, because that runtime is installed but it is not on the VS2019 image.

From Microsoft.NETCore.App 2.1.x onwards, the System.ComponentModel.EventBasedAsync.dll the assembly version is 4.1.1.0.

**Comment by skwasjer** [comment]
Ignore my last, it would require significant refactor due to async nature of HttpClient.

**Comment by Jericho** [comment]
I confirm that I am also [seeing this issue](https://ci.appveyor.com/project/Jericho/http-multipart-data-parser/builds/29074972#L32) with GitVersion.CommandLine [version 5.1.2](https://github.com/Http-Multipart-Data-Parser/Http-Multipart-Data-Parser/blob/develop/build.cake#L5). Another of my projects is using GitVersion.CommandLine [version 5.0.1](https://github.com/Jericho/StrongGrid/blob/develop/build.cake#L5) and is [not experiencing this problem](https://ci.appveyor.com/project/Jericho/stronggrid#L31).

Also, I can confirm that I am not experiencing this problem when running my build script locally:
```
========================================                                                                                                            
AppVeyor-Build_Number                                                                                                                               
========================================                                                                                                            
Executing task: AppVeyor-Build_Number                                                                                                               
Executing: "D:/_build/Http-Multipart-Data-Parser/tools/GitVersion.CommandLine.5.1.2/tools/GitVersion.exe" -output buildserver -verbosity Debug      
INFO [11/23/19 17:04:23:08] Working directory: D:\_build\Http-Multipart-Data-Parser                                                                 
INFO [11/23/19 17:04:23:09] IsDynamicGitRepository: False                                                                                           
INFO [11/23/19 17:04:23:11] Returning Project Root from DotGitDirectory: D:\_build\Http-Multipart-Data-Parser\.git\ - D:\_build\Http-Multipart-Data-
Parser\                                                                                                                                             
INFO [11/23/19 17:04:23:11] Running on Windows.                                                                                                     
INFO [11/23/19 17:04:23:11] Project root is: D:\_build\Http-Multipart-Data-Parser\                                                                  
INFO [11/23/19 17:04:23:11] DotGit directory is: D:\_build\Http-Multipart-Data-Parser\.git                                                          
INFO [11/23/19 17:04:23:15] Begin: Loading version variables from disk cache                                                                        
  INFO [11/23/19 17:04:23:16] Begin: Deserializing version variables from cache file D:\_build\Http-Multipart-Data-Parser\.git\gitversion_cache\13BE
269124B2E5D159064440AA6A034F4A59A4A0.yml                                                                                                            
  INFO [11/23/19 17:04:23:21] End: Deserializing version variables from cache file D:\_build\Http-Multipart-Data-Parser\.git\gitversion_cache\13BE26
9124B2E5D159064440AA6A034F4A59A4A0.yml (Took: 58.71ms)                                                                                              
  INFO [11/23/19 17:04:23:22] End: Loading version variables from disk cache (Took: 61.41ms)                                                        
Finished executing task: AppVeyor-Build_Number
```

**Comment by arturcic** [comment]
@Jericho is this a public repo?

**Comment by Jericho** [comment]
@arturcic yes: https://github.com/Http-Multipart-Data-Parser/Http-Multipart-Data-Parser

**Comment by arturcic** [comment]
@joaosantana @Caleb9 @skwasjer @Jericho , can you guys test version 5.1.3-beta.1+28? and let me know if it works for you

**Comment by Caleb9** [comment]
@arturcic I just tried it. Still doesn't work but with different error now:

```
gitversion /verbosity warn /l console /output buildserver /nofetch /b %APPVEYOR_REPO_BRANCH%
Executing GenerateSetVersionMessage for 'AppVeyor'.
  ERROR [11/25/19 12:51:18:60] An unexpected error occurred:
System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'System.Net.Http.HttpClient'.
   at System.Net.Http.HttpClient.CheckDisposed()
   at System.Net.Http.HttpClient.SendAsync(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.PutAsync(Uri requestUri, HttpContent content, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.PutAsync(Uri requestUri, HttpContent content)
   at System.Net.Http.HttpClient.PutAsync(String requestUri, HttpContent content)
   at GitVersion.BuildServers.AppVeyor.GenerateSetVersionMessage(VersionVariables variables)
   at GitVersion.BuildServerBase.WriteIntegration(Action`1 writer, VersionVariables variables)
   at GitVersion.ExecCommand.Execute()
   at GitVersion.GitVersionExecutor.VerifyArgumentsAndRun(Arguments arguments)
  ERROR [11/25/19 12:51:18:60] An unexpected error occurred:
System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'System.Net.Http.HttpClient'.
   at System.Net.Http.HttpClient.CheckDisposed()
   at System.Net.Http.HttpClient.SendAsync(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.PutAsync(Uri requestUri, HttpContent content, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.PutAsync(Uri requestUri, HttpContent content)
   at System.Net.Http.HttpClient.PutAsync(String requestUri, HttpContent content)
   at GitVersion.BuildServers.AppVeyor.GenerateSetVersionMessage(VersionVariables variables)
   at GitVersion.BuildServerBase.WriteIntegration(Action`1 writer, VersionVariables variables)
   at GitVersion.ExecCommand.Execute()
   at GitVersion.GitVersionExecutor.VerifyArgumentsAndRun(Arguments arguments)
```

**Comment by arturcic** [comment]
@Caleb9  which version of the VM image are you using? as GitVersion itself builds with Visual Studio 2019 version and it works without errors

**Comment by Caleb9** [comment]
@arturcic With "Visual Studio 2019" image. However [according to docs](https://www.appveyor.com/docs/windows-images-software/#tools) GitVersion is not available on this image.

**Comment by arturcic** [comment]
well on Appveyor it's installed a really old version, you should manually install the latest, as we don't have resources to support that old version. So I would recommed to use the latest version of the VM image if possible

**Comment by Caleb9** [comment]
How do you mean? Are there different versions of "Visual Studio 2019" image? Otherwise if you mean GitVersion then I just tried installing 5.1.3-beta1-28 as you suggested.

**Comment by Caleb9** [comment]
As far as I understand it if my appveyor.yml script contains
```
image: Visual Studio 2019
```
it'll use the latest available Visual Studio 2019 image, which seem to be from 8th of November as of today: https://www.appveyor.com/updates/

**Comment by arturcic** [comment]
Can you try again with this version 5.1.3-beta.1+29?

**Comment by Caleb9** [comment]
Yes, just tried it and it doesn't show any errors, so great.

I still have an issue with AppVeyor not being able to resolve environment variable by `$(GitVersion_SemVer)` even though GitVersion's output says it has been added. But that might be caused by something else, I need to investigate more and I'll report my findings.

Anyway thanks for your effort @arturcic, hope I can resolve the other issue until stable 5.1.3 comes out :).

**Comment by arturcic** [comment]
Glad it helped. Thank you for your help with the feedback 👍 
@joaosantana @skwasjer @Jericho, can you also confirm 5.1.3-beta.1+29 works for you?

**Comment by Jericho** [comment]
@arturcic I just test beta 1.28 and, as @Caleb9 described earlier, the error is not different: [System.ObjectDisposedException: Cannot access a disposed object. Object name: System.Net.Http.HttpClient](https://ci.appveyor.com/project/Jericho/http-multipart-data-parser/builds/29103890#L30).

Give me minute I'll try beta 1.29...

**Comment by Jericho** [comment]
@arturcic Good news: I confirm that beta 1.29 solves the problem as you can [see here](https://ci.appveyor.com/project/Jericho/http-multipart-data-parser/builds/29104079#L32).

**Comment by arturcic** [comment]
@Jericho thanks for the feedback, I have now 2 confirmations, I'll close the issue

**Comment by Jericho** [comment]
@arturcic I just observed a new problem that started when I upgraded to GitVersion beta 1.29: the build version in AppVeyor is no longer updated. As you can see in this screenshot, my version number when from `3.1.0 Alpha` to `1.0.x`:

![image](https://user-images.githubusercontent.com/112710/69907271-f9ce5c00-139f-11ea-9abd-119d16e9f292.png)

See here: https://ci.appveyor.com/project/Jericho/http-multipart-data-parser/history

**Comment by arturcic** [comment]
@Jericho I had a look in the logs https://ci.appveyor.com/project/Jericho/http-multipart-data-parser/builds/29227223#L59, looks like it is trying to set the build number with version 3.1.0-alpha, but I'm not sure why it doesn't change in the appveyor, we need more investigations

**Comment by Mpdreamz** [comment]
Seeing the same error on: 

https://github.com/serilog/serilog-sinks-elasticsearch/pull/300

https://ci.appveyor.com/project/serilog/serilog-sinks-elasticsearch/builds/29260099

Will https://chocolatey.org/packages/GitVersion.Portable be updated with this fix?

**Comment by jameswoodley** [comment]
I'm now broken due to this too. I am using the chocolatey package much like @Mpdreamz on Appveyor. Any chance this will be resolved asap, my builds are now broken

**Comment by arturcic** [comment]
@Mpdreamz @jameswoodley do you mind trying version 5.1.3-beta.1+59. There is still an issue with not updating the build number, but it does not fail the build

**Comment by arturcic** [comment]
https://github.com/appveyor/ci/issues/3228

**Comment by nvborisenko** [comment]
Fixed in 5.1.3-beta1.89
